pkgs/development/python-modules/langgraph/default.nix

```diff
@@ -46,10 +51,15 @@ buildPythonPackage rec {
     dataclasses-json
     grandalf
     httpx
+    langgraph-checkpoint
```

Comment:
`langgraph-checkpoint` should go to `dependencies`.
https://github.com/langchain-ai/langgraph/blob/a16a33806c60e80be1fb64b54f2e5835f6080c01/libs/langgraph/pyproject.toml#L10-L13

Reply:
Fixed.